plugins/inputs/riemann_listener/riemann_listener.go

```diff
@@ -170,8 +170,7 @@ func (rsl *riemannListener) read(conn net.Conn) {
 
 	for {
 		if rsl.ReadTimeout != nil && rsl.ReadTimeout.Duration > 0 {
-
-			err = conn.SetDeadline(time.Now().Add(rsl.ReadTimeout.Duration))
+			conn.SetDeadline(time.Now().Add(rsl.ReadTimeout.Duration))
```

Comment:
do we log the error instead?

Reply:
Nope, there are few other occurences of `conn.SetDeadline(...)` in Telegraf and not even one check this error returned. 

@ssoroka What do you suggest?